app/src/main/java/io/github/mattpvaughn/chronicle/features/player/AudiobookMediaSessionCallback.kt

```diff
@@ -122,6 +123,13 @@ class AudiobookMediaSessionCallback @Inject constructor(
         currentPlayer.seekRelative(trackListStateManager, SKIP_BACKWARDS_DURATION_MS_SIGNED)
     }
 
+    private fun changeSpeed() {
+        changeSpeed(prefsRepo)
+        Timber.i("New Speed: %s", prefsRepo.playbackSpeed)
+        Toast.makeText(appContext, "New Speed: " + prefsRepo.playbackSpeed, Toast.LENGTH_SHORT)
+            .show()
```

Comment:
Is there another way we could convey playback speed to users? I know the screenshot posted for on [the issue](https://github.com/mattttvaughn/chronicle/issues/4) shows the playback speed on the icon, not sure about the difficulty of that

Alternatively, if there isn't an easy way to do the above, using a fast forward icon instead seems a little more intuitive for a user who we shouldn't expect to read a 5-10s toast

Reply:
Yeah I gave it a little bit of time to try and figure that out. I added the button since I'm heading on a road trip and wanted the option. When I get back I'll probably do some more digging to try and figure it out.